setup.py

```diff
@@ -12,7 +12,7 @@
         "Programming Language :: Python :: 3",
         "Operating System :: OS Independent",
     ],
-    py_modules=['jinahub.image.encoder.torch_encoder'],
+    py_modules=['jinahub.image.encoder.torch_encoder', 'jinahub.image.encoder.models'],
```

Comment:
why this?

Reply:
We need this module installed. It contains relevant code for the executor.